build/osx/OSConsX.py

```diff
@@ -614,7 +615,7 @@ def do_codesign(target, source, env):
 
     keychain = env.get("CODESIGN_KEYCHAIN", None)
     keychain_password = env.get("CODESIGN_KEYCHAIN_PASSWORD", None)
-    installer_identity = env.get("CODESIGN_INSTALLER_IDENTITY", None)
+    # installer_identity = env.get("CODESIGN_INSTALLER_IDENTITY", None)
```

Comment:
There should be a note why this code has been commented out and not deleted

Reply:
same here